bower.json

```diff
@@ -0,0 +1,27 @@
+{
+	"name": "Dialer",
+	"version": "1.0.0",
+	"homepage": "https://github.com/fxos/dialer",
+	"authors": [
+		"Doug Sherk <github@sherk.me>"
+	],
+	"description": "Replaceable Dialer for Firefox OS.",
+	"main": "app.js",
+	"license": "MIT",
+	"private": true,
+	"ignore": [
+		"**/.*",
+		"node_modules",
+		"bower_components",
+		"app/components",
+		"test",
+		"tests"
+	],
+	"dependencies": {
+		"fxos-mvc": "git@github.com:fxos/mvc.git",
+		"gaia-theme": "gaia-components/gaia-theme",
+		"gaia-header": "gaia-components/gaia-header",
+		"gaia-button": "gaia-components/gaia-button",
+		"gaia-fonts": "gaia-components/gaia-fonts"
+	}
```

Comment:
Agreed! I feel like github prefers spaces and tries to punish people for using tabs by making the width 8. Fucking @github (just kidding, love you guys).

I've found 2 ways you can make things more pretty. One, append the `ts` query string parameter, like so:
https://github.com/fxos/dialer/pull/1/files?ts=2

Or use an add on like stylish to modify the stylesheet:
https://addons.mozilla.org/en-US/firefox/addon/stylish/
```
pre {
  tab-size: 2;
}